script/deploy-canary-demo

```diff
@@ -9,7 +9,11 @@ cd $GITHUB_WORKSPACE/demos/serverless
 npm ci
 
 echo "Deploying to alpha stage for canary"
-npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -i eu-south-1 -t -l
+npm run deploy -- -b chime-sdk-demo-canary -s chime-sdk-demo-canary -o chime-sdk-demo-canary -u false -i eu-south-1 -t -l
+npm run deploy -- -r us-east-1 -b chime-sdk-demo-canary-iad -s chime-sdk-demo-canary-iad -o chime-sdk-demo-canary-iad -c $CHIME_ENDPOINT_IAD -t -l
```

Comment:
Why do you need to pass in endpoint for this? It should be enough to just pass in region right?